backend/Bindings/Edna.Bindings.LtiAdvantage/Utils/EdnaExternalHttpHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Net.Security;
+
+namespace Edna.Bindings.LtiAdvantage.Utils
```

Comment:
@adbindal I had to create a duplicate file for LTIAdvantage. Adding a dependency from `LTIAdvantageBinding` on the other `Utils` project was causing a package version conflict issue (for `System.IdentityModel.TokensJwt`). The version is 6.5.1 and 6.6.0 in the two, respectively.

Reply:
Hmm. Sounds interesting. Should we consider upgrading those instead 🤔 ?

Reply:
I wasn't sure if this was done on purpose, or because of the gap between development periods. The two versions were released pretty close in time though, so that could be a possibility. Should we take that as a separate task, so that we can keep track in case it breaks things up?

Reply:
Sure. Let's plan a follow-up. Having it in LtiAdvantage is as confusing as it sounds.